lib/models/os.ts

```diff
@@ -30,6 +30,7 @@ const BALENAOS_VERSION_REGEX = /v?\d+\.\d+\.\d+(\.rev\d+)?((\-|\+).+)?/;
 export interface ImgConfigOptions {
 	network?: 'ethernet' | 'wifi';
 	appUpdatePollInterval?: number;
+	provisioningKeyName?: string;
```

Comment:
Let's also add a test case to test that `getConfig` also sets the name that we are providing to the generated provisioning api key.

Reply:
I added a `provisioningKeyName` for the `getConfig` spec, but we don't get back the key name only the actual key. I tested this functionality in the core-api here: https://github.com/balena-io/open-balena-api/pull/716/files#diff-1880e5d78ef6ce81f0e72c9bf111a7783e5dd39c5de9278026b13fae90104f4dR81

Should you see a way to access the key object on the SDK spec side?